packages/vuetify/src/directives/ripple/_variables.scss

```diff
@@ -1,6 +1 @@
 @import '../../styles/styles.sass';
-
-$ripple-animation-transition-in: transform .25s map-get($transition, 'fast-in-slow-out'),
-                                 opacity .1s map-get($transition, 'fast-in-slow-out') !default;
-$ripple-animation-transition-out: opacity .3s map-get($transition, 'fast-in-slow-out') !default;
-$ripple-animation-visible-opacity: .15 !default;
```

Comment:
Prever this file over the global one if the variables aren't used anywhere else.

Reply:
I've thought about that. but the problem is that it won't be documented! if it's written only in ripple `variable.scss`.
directives are not mentioned in [Sass variables page](https://vuetifyjs.com/en/features/sass-variables/#variable-api)

If I keep it in global variable it'll be added to `$vuetify`  object

Reply:
Ok remove this file entirely then if it does nothing.

Reply:
Done  👍